### PR TITLE
Allow setting name for custom/headless outputs

### PR DESCRIPTION
**Pull request by phonetic112**
Title. For some reason the compiler gave errors when I tried to get the return of `wlr_x11_output_create` so I can't name the x11 outputs I guess.

Example:

```
$ hyprctl output create headless hello
Monitor WL-1 (ID 0):
	1183x1026@144.00000 at 0x0
	description: 
	make: 
	model: 
	serial: 
	active workspace: 1 (1)
	special workspace: 0 ()
	reserved: 0 0 0 0
	scale: 1.00
	transform: 0
	focused: yes
	dpmsStatus: 1
	vrr: 1
	activelyTearing: false
	disabled: false
	currentFormat: XRGB8888
	availableModes: 

Monitor hello (ID 1):
	1920x1080@144.00000 at 1183x0
	description: 
	make: 
	model: 
	serial: 
	active workspace: 2 (2)
	special workspace: 0 ()
	reserved: 0 0 0 0
	scale: 1.00
	transform: 0
	focused: no
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false
	disabled: false
	currentFormat: XRGB8888
	availableModes: 
```

### CODE REVIEW

**Comment by vaxerski** [comment]
x11 backend is disabled, thats why

File: src/debug/HyprCtl.cpp [hunk 1]
```diff
@@ -1374,73 +1374,65 @@ std::string decorationRequest(eHyprCtlOutputFormat format, std::string request)
...
-
-    if (DATA->first.empty() || DATA->first == "auto") {
-        if (wlr_backend_is_wl(backend)) {
-            wlr_wl_output_create(backend);
-            DATA->second = true;
-        } else if (wlr_backend_is_x11(backend)) {
+    if (type.empty() || type == "auto") {
+        if (wlr_backend_is_wl(backend))
+            output = wlr_wl_output_create(backend);
+        else if (wlr_backend_is_x11(backend))
```
Comment (vaxerski) [inline src/debug/HyprCtl.cpp:1383]: while we're at it, we can nuke all x11 things (disabled at comptime anyways)

Comment (phonetic112) [reply src/debug/HyprCtl.cpp]: done

File: src/layout/IHyprLayout.cpp [hunk 2]
```diff
@@ -147,7 +147,7 @@ void IHyprLayout::onWindowCreatedFloating(PHLWINDOW pWindow) {
                 pWindow->m_vRealPosition = pWindow->m_pXDGSurface->toplevel->parent->window->m_vRealPosition.goal() +
                     pWindow->m_pXDGSurface->toplevel->parent->window->m_vRealSize.goal() / 2.F - desiredGeometry.size() / 2.F;
             else
-                pWindow->m_vRealPosition = PMONITOR->vecPosition + desiredGeometry.size() / 2.F;
+                pWindow->m_vRealPosition = PMONITOR->vecPosition + PMONITOR->vecSize / 2.F - desiredGeometry.size() / 2.F;
```
Comment (vaxerski) [inline src/layout/IHyprLayout.cpp:150]: unintentional? rebase on main

Comment (phonetic112) [reply src/layout/IHyprLayout.cpp]: should be fixed

**Review by vaxerski — APPROVED** [review]
thanks!

**Pull request merged**